promscale/send-data/jaeger.md

```diff
@@ -0,0 +1,51 @@
+# Send Jaeger traces to Promscale
+Promscale natively supports the OpenTelemetry Line Protocol (OTLP) for traces.
+To ingest Jaeger traces to Promscale, use the OpenTelemetry Collector.
+OpenTelemetry Collector converts the Jaeger traces to OpenTelemetry traces.
+
+# Send data using the OpenTelemetry Collector
+You can configure OpenTelemetry Collector to forward Jaeger traces to Promscale
+using [Jaeger receiver][jaeger-receiver] and OpenTelemetry exporter.
+
+Here's an example configuration file for OpenTelemetry Collector to forward
+traces to Promscale after batch processing the data.
+
+```yaml
+receivers:
+  jaeger:
+    protocols:
+      grpc:
+      thrift_binary:
+      thrift_compact:
+      thrift_http:
+exporters:
+  otlp:
+    endpoint: "<PROMSCALE_HOST>:<gRPC_PORT>"
+    tls:
+      insecure: true
+    sending_queue:
+      queue_size: 1000000
+    timeout: 10s
+processors:
+  batch:
+    send_batch_size: 4000
+    send_batch_max_size: 4000
+    timeout: 10s
+service:
+  pipelines:
+    traces:
+      receivers: [jaeger]
+      exporters: [otlp]
+      processors: [batch]
+```
+
+Where: 
+* `<PROMSCALE_HOST>`: hostname of Promscale.
+* `<gRPC_PORT>`: gRPC port of Promscale. The default port is 9202.
```

Comment:
```suggestion
Use these variables:
* `<PROMSCALE_HOST>`: hostname of Promscale.
* `<gRPC_PORT>`: gRPC port of Promscale. The default port is 9202.
```

Reply:
@Loquacity "Use these variables" sounds a bit strange since it seems we are asking the reader to indeed define a variable. Rather than variables, those are placehoders that we want to the user to replace. Could you suggest alternate wording?

Reply:
Replace these variables?